test/test_site/expected/bugs/index.html

```diff
@@ -6,13 +6,13 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <title>Open Bugs</title>
-    <link rel="stylesheet" href="..\markbind\css\bootstrap.min.css">
-    <link rel="stylesheet" href="..\markbind\css\bootstrap-vue.min.css">
-    <link rel="stylesheet" href="..\markbind\css\font-awesome.min.css" >
-    <link rel="stylesheet" href="..\markbind\css\bootstrap-glyphicons.min.css" >
-    <link rel="stylesheet" href="..\markbind\css\github.min.css">
-    <link rel="stylesheet" href="..\markbind\css\markbind.css">
-    <link rel="stylesheet" href="..\markbind\layouts\default\styles.css">
+    <link rel="stylesheet" href="../markbind/css/bootstrap.min.css">
+    <link rel="stylesheet" href="../markbind/css/bootstrap-vue.min.css">
+    <link rel="stylesheet" href="../markbind/css/font-awesome.min.css" >
+    <link rel="stylesheet" href="../markbind/css/bootstrap-glyphicons.min.css" >
+    <link rel="stylesheet" href="../markbind/css/github.min.css">
+    <link rel="stylesheet" href="../markbind/css/markbind.css">
+    <link rel="stylesheet" href="../markbind/layouts/default/styles.css">
```

Comment:
We should be consistent with the slash direction, so let's stick to `\`.

Reply:
We should use `/`, especially for URLs. Is `\` generated by Windows here?

Reply:
Yes `\` is generated by Windows. We currently update test files by copying over content from `_site` to `expected`, so the slashes are going to change depending on the OS that the person is using when he/she updates the tests, which is not really ideal...

Maybe code-wise we should standardize how MarkBind output slashes for file paths (can be a separate PR for that).

Reply:
Agreed.

Reply:
@acjh @yamgent could I check if I should change the slash direction in my PR, or should we leave it to be fixed in another PR?

Thanks for looking into it!

Reply:
> could I check if I should change the slash direction in my PR, or should we leave it to be fixed in another PR?

Those will be dealt with by the person that is resolving https://github.com/MarkBind/markbind/issues/628.